### PR TITLE
feat: added support for creating a new wallet

### PR DESCRIPTION
**Pull request by frazarshad**

### CODE REVIEW

File: commands/keplr.js [hunk 1]
```diff
@@ -93,6 +93,88 @@ const keplr = {
...
+      onboardingElements.phraseCount24,
+      playwright.keplrWindow(),
+    );
+    await playwright.waitAndClickByText(
+      onboardingElements.copyToClipboardButton,
+      playwright.keplrWindow(),
+    );
+
+    const passPhraseArray = clipboardy.readSync().split(' ');
+    await playwright.waitAndClickByText(
```
Comment (rabi-siddique) [inline commands/keplr.js:123]: Nit: `mnemonicPhraseArray`

File: commands/keplr.js [hunk 2]
```diff
@@ -93,6 +93,88 @@ const keplr = {
...
+      onboardingElements.nextButton,
+      playwright.keplrWindow(),
+    );
+
+    const wordLabels = await playwright.waitForElementsByText(
+      /Word #\d+/,
+      playwright.keplrWindow(),
```
Comment (rabi-siddique) [inline commands/keplr.js:130]: `/Word #\d+/` is already quite descriptive. Although, if we want to be more specific about the range of digits, we can use: `/Word #([1-9]|1[0-9]|2[0-4])/`

File: commands/playwright-keplr.js [hunk 3]
```diff
@@ -194,6 +194,19 @@ module.exports = {
     }
     return element;
   },
+  async waitForElementsByText(text, page = keplrWindow, exact = false) {
+    await module.exports.waitUntilStable(page);
+    const elements = await page.getByText(text, { exact: exact }).all();
+    await Promise.all(elements.map(element => element.waitFor()));
+    if (process.env.STABLE_MODE) {
```
Comment (rabi-siddique) [inline commands/playwright-keplr.js:201]: Why do we need this: `await Promise.all(elements.map(element => element.waitFor()));`?

**Review by rabi-siddique — APPROVED** [review]

File: commands/keplr.js (hunk 2 above)
Comment (frazarshad) [reply commands/keplr.js:130]: i think this would reduce readibilty quite a bit

**Pull request merged**